### PR TITLE
How to Get Started with YARA for Malware Analysis

### LINKED ISSUE #6976: [Security]Mastering YARA for Malware Analysis the right way.
### Topic Suggestion
Mastering YARA for Malware Analysis the right way.

### Writing sample(s): 
https://www.section.io/engineering-education/getting-started-with-visual-basic-net/

#### Proposed title of article
Mastering YARA for Malware Analysis the right way. 

#### Proposed article introduction
YARA is a tool used by cyber security Engineers for malware analysis. It provides a rule-based approach to detect patterns of malware characteristics in files. A rule is a basic description of malware characteristics. The malware contains specific characteristics, such as unique strings, which we can use to identify if a file has malware.

#### Key takeaways 
- What is YARA
- How to install YARA in both Linux and Windows Systems.
- YARA Rules Syntax 
- Running YARA Rules

#### Article quality
This article will guide the user on how to install YARA, its basic syntax and go a step further and demonstrate how YARA rules can be used in malware analysis .

### PR DESCRIPTION
**Pull request by Dyrstiu**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #6976

### CODE REVIEW

**Comment by hectorkambow** [comment]
@section-engineering-education/peer-reviewers

File: content/articles/mastering-yara-for-malware-analysis-the-right-way/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+*(Source: [av-test, 2022](https://www.av-test.org/en/statistics/malware/))*
+
+Malicious programs, commonly known as malware, can be defined as software developed intentionally to harm computer systems or alter their normal mode of operation. We can spread malware through various methods, such as downloads from pirated sites, storage media, i.e. USB drives, and emails with malicious links and attachments that appear to be from legitimate sources, among others.
+
+Computer systems infected with malware often display some signs. You may notice that your computer is getting slower, settings on some of your applications have been altered, some files are missing, or existence of new unknown files among other signs.
+
+In this article, we look at YARA, a malware analysis tool used to detect patterns of malware characteristics in files, using a rule-based approach. We will focus on What YARA is, How to install it in windows and Linux environments, and finally handle its syntax.
+
```
Comment (Jethro-magaji) [inline content/articles/mastering-yara-for-malware-analysis-the-right-way/index.md:9]: Add a "Table of contents" and "Prerequisites" section here
Check out this [guide](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

File: content/authors/felix-vaati/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Felix Vaati
 type: authors
 interests: PHP & Kotlin
-twitter: https://twitter.com/DyrstiuAppdh
+twitter: https://twitter.com/Dyrstiu
 images:
   - url: /engineering-education/authors/felix-vaati/avatar.png
```
Comment (Jethro-magaji) [inline content/authors/felix-vaati/index.md:7]: You haven't added your profile image to the `authors` folder.

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi! @Dyrstiu 
Well done for such an amazing article. However, there are some corrections you need to make.
Please, run your article on tools like Grammarly and ProWritingAid for better editing. This would save the review time to review your work.

Here's a [link](https://github.com/section-engineering-education/engineering-education/pull/6210/commits/acc44893d4ec258f4e077b9ce8e3e2ca2b9d6a9f) to a PR with little or no revision to guide you on formatting. If you have any questions or need help, you can reach out to me on the EngEd Slack channel.
Thanks.

File: content/authors/felix-vaati/index.md (hunk 2 above)
Comment (Dyrstiu) [reply content/authors/felix-vaati/index.md:7]: Hi , the image was already there. What i changed was the twitter URL only as you can see from github snippet above

File: content/articles/mastering-yara-for-malware-analysis-the-right-way/index.md (hunk 1 above)
Comment (Dyrstiu) [reply content/articles/mastering-yara-for-malware-analysis-the-right-way/index.md:9]: @Jethro-magaji I have updated the structure of the article.

**Comment by louisefindlay23** [comment]
Plagiarism check complete

**Comment by hectorkambow** [comment]
@Jethro-magaji can we get a status on this  PR?

**Comment by Jethro-magaji** [comment]
> @Jethro-magaji can we get a status on this  PR? Thank you

Currently working on it...

**Comment by hectorkambow** [comment]
@Jethro-magaji  

Noted thank you - if any more edits are needed we should be closing this PR.

**Comment by Dyrstiu** [comment]
@Jethro-magaji any updates 🤔

**Review by Jethro-magaji — APPROVED** [review]
Hello @Dyrstiu 
Well done for coming up with this article. Please take note of these corrections in case of next time.
Thanks.

**Comment by Dyrstiu** [comment]
@Jethro-magaji 
I have taken note of the corrections. Thank you

**Comment by hectorkambow** [comment]
jumping on this one

**Comment by hectorkambow** [comment]
👏 
great work @Dyrstiu 
Be sure to see your live published article on the site (it should be live within a 10-30 mins).

To give your article a boost in page engagement - share it across your social site (linkedin/twitter) and tag the EngEd program so we can also share it and give it some traction as well!

>_**If you have not done so already - consider adding a skill section to your author page. This will go a long way for potential future employers.**_

**Pull request merged**